bundle.js

```diff
@@ -568,14 +568,14 @@ var accname = (function (exports) {
      * @param elem - the function checks if 'elem' allows name from content
      * @return - true if elem allows name from content, false otherwise
      */
-    function allowsNameFromContent(elem) {
+    function roleAllowsNameFromContent(elem) {
```

Comment:
The roles from list3 also (sometimes) allows name from content. Maybe add `always` in there?